Sources/AsyncQueue/FIFOQueue.swift

```diff
@@ -54,7 +54,7 @@ public final class FIFOQueue: Sendable {
     /// Schedules an asynchronous task for execution and immediately returns.
     /// The scheduled task will not execute until all prior tasks – including suspended tasks – have completed.
     /// - Parameter task: The task to enqueue.
-    public func async(_ task: @escaping @Sendable () async -> Void) {
+    public func enqueue(_ task: @escaping @Sendable () async -> Void) {
```

Comment:
Do we need to edit this `await` in this line? https://github.com/dfed/swift-async-queue/blob/3454ea6497aeb917b6fe084efd69d95d9c7cff35/Sources/AsyncQueue/FIFOQueue.swift#L25

Reply:
oooooh solid find!